solana-programs/claimable-tokens/cli/src/main.rs

```diff
@@ -468,8 +468,8 @@ fn main() -> anyhow::Result<()> {
             let _base_account = (|| -> anyhow::Result<_> {
                 let mint = pubkey_of(args, "mint").unwrap();
                 let program_id = pubkey_of(args, "program_id").unwrap();
-                println!("Recieved mint {:?}", mint);
-                println!("Recieved program_id {:?}", program_id);
```

Comment:
Yeah that's definitely a typo alright.

Unfortunately, the bar for a new deploy of the program is quite high, and it's important that this code matches what code is currently deployed on the production program deploy. With that in mind, I think we're stuck with this typo until a change that meets the bar and requires us to deploy the program to a new address, in which case we can bundle this change in with that new deploy.

Hopefully that makes sense!